exercise3/multiply.js

```diff
@@ -0,0 +1,3 @@
+function multiply(number1, number2) {
+  return number1 * number2;
+}
```

Comment:
`SUGGESTION` Hello @devfabien I feel if you change the function into an arrow function and use implicit return would be a great idea. But the above solution is still good. 👍

Reply:
Hello @gasangw am glad for this comment but to be consistency in my codes I prefer using normal functions in all exercises